modules/ROOT/pages/mule-deployment-model.adoc

```diff
@@ -4,13 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint studio, esb, deploy, architecture
 
-A Mule Project for deployment is a *Mule Application*. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
+A Mule project for deployment is a *Mule Application*. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
```

Comment:
I am not sure we should be capitalizing Application in "Mule Application." I would say no, but I suggest checking the rest of the doc and maybe checking with Duke on this. I know Mysti things we capitalize too many things.

Reply:
Updated.